From 23c3a7a624b2f541d174cc807eddece0d610951b Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 13 Aug 2018 06:56:35 +0300
Subject: [PATCH] BTL: update comment on last patch

Note: This is no good for release
We do need equalization redo!

commit 2357373a9050196dce19760580bc9441a8af1749
Author: Igor Grinberg <igrinberg@habana.ai>
Date:   Sun Aug 12 11:50:07 2018 +0300

    btl: pcie: adjust the GEN3 registeres
---

diff --git a/board/habanalabs/common/btl/pcie_phy.s b/board/habanalabs/common/btl/pcie_phy.s
index dfedd0d..f6f60f2 100644
--- a/board/habanalabs/common/btl/pcie_phy.s
+++ b/board/habanalabs/common/btl/pcie_phy.s
@@ -37,7 +37,8 @@
 cfg mmPCIE_PHY_PROTOCOL0_EXT_TX_VBOOST_LVL, 0x00000004
 cfg mmPCIE_PHY_PHY_EXT_CTRL_SEL, 0x00000001
 
-/* disable the EQ */
+/* disable the EQ REDO */
+/* NOTE:::::::::::::::::: this is BAD do not release with this */
 cfg mmPCIE_DBI_GEN3_RELATED_OFF, 0x00402801
 
 /*
